db/connect.js

```diff
@@ -0,0 +1,12 @@
+const mongoose = require('mongoose')
+
+// http://mongoosejs.com/docs/promises.html
+mongoose.Promise = global.Promise
+
+const dbUrl = 'mongodb://localhost/testdb'
```

Comment:
Do we want to set up an process.env.PRODUCTION variable now to connect to different databases?

Or shall we just do this for now.

I think maybe just do this for now actually since it's not hard to put things in an IF statement